tasks/es/ine.py

```diff
@@ -1869,37 +1874,82 @@ def columns(self):
         source = input_['source']['ine-source']
         session = current_session()
         total_pop = input_['seccion_columns']['total_pop'].get(session)
-        columns = OrderedDict([
-            ('gender', OBSColumn(
-                type='Text',
-                name='Gender',
-                weight=0
-            ))
-        ])
-        for i in xrange(0, 20):
-            start = i * 5
-            end = start + 4
-            _id = 'pop_{start}_{end}'.format(start=start, end=end)
-            columns[_id] = OBSColumn(
-                type='Numeric',
-                name='Population age {start} to {end}'.format(
-                    start=start, end=end),
-                targets={total_pop: DENOMINATOR},
-                description='',
-                aggregate='sum',
-                weight=3,
-                tags=[spain, tags['age_gender'], units['people'], license, source]
-            )
-        columns['pop_100_more'] = OBSColumn(
+        columns = OrderedDict()
+
+        genders = [None, MALE, FEMALE]
+
+        total_pop_male = OBSColumn(
+            id='total_pop_male',
             type='Numeric',
-            name='Population age 100 or more'.format(
-                start=start, end=end),
+            name='Total population (male)',
```

Comment:
Uhm, we have this column, with another name (`Males`) from INE data. How can we solve this duplication?

Ideas:
- Use two columns. Bad user experience.
- Use the same id for both columns. Since they have different boundaries, there is no conflict for now, could be in the future. If there is conflict, we would have to make sure the value in both datasets is the same.
- Mark the new columns with no tags. That should make them disappear from numerators, but appear on denominators (untested)

Rereading the code, it seems we are already doing approach 2,and it seems reasonable enough. Does that sound good?

Reply:
Using no tags on the new columns makes them dissapear both from numerators and denominators (tested).
The current approach sounds reasonable enough for me.

Reply:
Ok, then let's go for it.